x/launch/types/chain.go

```diff
@@ -9,7 +9,10 @@ import (
 	codec "github.com/cosmos/cosmos-sdk/codec/types"
 )
 
-const chainIDSeparator = "-"
+const (
+	ChainIDSeparator   = "-"
+	ChainNameMaxLength = 30
```

Comment:
Is this const need to be public/exported?

Reply:
In order to use it in tests for edge case (trying to create a name with `ChainNameMaxLength +1`)